config.json

```diff
@@ -1067,6 +1067,15 @@
       "uuid": "0c17444e-65b0-4bc8-a7e1-ee6ab8c2e078"
     },
     {
+      "difficulty": 2,
+      "slug": "isbn-verifier",
+      "topics": [
+        "arrays"
+      ],
+      "unlocked_by": "difference-of-squares",
+      "uuid": "6b93edd9-0de4-4c80-6419-68b2f8232b91c3425fa"
+   },
```

Comment:
Check that spacing? 

One tool that I find helps tremendously to avoid spacing issues (and general non gofmt formatting issues) is [editorconfig](http://editorconfig.org/). Basically if you add an .editorconfig file to the root of your copy of the repo and configure your code editor it will automatically format files in a style independent of your default configuration. Great for adapting to the code styling of different repos. I use [the one from the main site repo](https://raw.githubusercontent.com/exercism/exercism.io/master/.editorconfig) and it works for things like the *.md or *.json files.

Thank goodness Go has gofmt :rocket: